dvc/config.py

```diff
@@ -103,6 +103,7 @@ class RelPath(str):
 REMOTE_COMMON = {
     "url": str,
     "checksum_jobs": All(Coerce(int), Range(1)),
+    "jobs": All(Coerce(int), Range(1)),
```

Comment:
With no defaults and fancy multiplications, the field is required and could be only `1 <=`.

Reply:
>  the field is required

That's actually false :slightly_smiling_face:  Unless `Required()` is used, every field is considered `Optional`. We are using `voluptuous` for Schema, which has such default settings. A bit confusing, I know :slightly_smiling_face: